tools/acorn-optimizer.mjs

```diff
@@ -1731,9 +1731,6 @@ function minifyLocals(ast) {
   assert(extraInfo && extraInfo.globals);
 
   for (const fun of ast.body) {
-    if (!fun.type === 'FunctionDeclaration') {
```

Comment:
How about `assert(fun.type == 'FunctionDeclaration')` just to be sure?

Reply:
Hmm, but that's not guaranteed at all? But I can see your point, the code below does seem to assume it's operating on function declarations. In that case `!==` is the correct fix after all.